config/inspectors.json

```diff
@@ -2274,5 +2274,30 @@
         "short_name": "Department of Veterans Affairs",
         "slug": "va",
         "statutory_reference": "5 U.S.C. App. 3 \u00A7 2(A)"
+    },
+    {
+        "active": true,
+        "agencies": ["Office of Special Counsel"],
+        "established_year": 1979,
+        "homepage_url": "http://www.osc.gov",
+        "hotline": [
+            {
+                "type": "url",
+                "value": "https://osc.gov/Pages/about.aspx"
+            },
+            {
+                "type": "phone",
+                "value": "800-572-2249"
+            },
+            {
+                "type": "mail",
+                "value": "Whistleblower Disclosure Hotline\n1730 M Street, N.W., Suite 218\nWashington, D.C. 20036-4505\n\nProhibited Personnel Practices\n1730 M Street, N.W., Suite 218\nWashington, D.C. 20036-4505"
+            }
+        ],
+        "name": "Office of Special Counsel",
+        "publishes_reports": true,
+        "short_name": "Office of Special Counsel",
+        "slug": "osc",
+        "statutory_reference": "5 U.S.C. \u00A7 1201"
```

Comment:
I think this should be section 1211, right? (see https://www.law.cornell.edu/uscode/text/5/1211)